variables.tf

```
@@ -33,6 +33,18 @@ variable "dead_letter_target_arn" {
   description = "The ARN of an SNS topic or SQS queue to notify when an invocation fails"
 }
 
+variable "destination_on_failure" {
+  description = "Amazon Resource Name (ARN) of the destination resource for failed asynchronous invocations"
+  type        = string
+  default     = null
+}
+
+variable "destination_on_success" {
+  description = "Amazon Resource Name (ARN) of the destination resource for successful asynchronous invocations"
+  type        = string
+  default     = null
```

```suggestion
  type        = string
  default     = null
  description = "ARN of the destination resource for successful asynchronous invocations"
```

We use ARN in other parts, so doesn't need the explanation, and sorted fields like other variables.